chainindexer.go

```diff
@@ -467,9 +468,13 @@ func (a *addrIndexer) indexBlockAddrs(blk *btcutil.Block) (database.BlockAddrInd
 				// Lookup and fetch the referenced output's tx.
 				prevOut := txIn.PreviousOutPoint
 				txList, err := a.server.db.FetchTxBySha(&prevOut.Hash)
-				if err != nil || len(txList) == 0 {
-					adxrLog.Errorf("Couldn't get referenced "+
-						"txOut (%v): %v", prevOut, err)
+				if len(txList) == 0 {
+					return nil, fmt.Errorf("tx: %v does not exist",
```

Comment:
"transaction %v not found".  It might exist on someone else's machine.  Also no colon here.